File: php/src/MBPay/Types/BalanceResponse.php
<?php

namespace MBPay\Types;

/**
 * 余额查询响应
 */
class BalanceResponse
{
    /**
     * @var int 可用余额（MB）
     */
    private $balance;

    /**
     * @var int 冻结余额（MB）
     */
    private $frozen;

    /**
     * @param int $balance
     * @param int $frozen
     */
    public function __construct(int $balance, int $frozen)
    {
        $this->balance = $balance;
        $this->frozen = $frozen;
    }

    /**
     * 获取可用余额
     *
     * @return int
     */
    public function getBalance(): int
    {
        return $this->balance;
    }

    /**
     * 获取冻结余额
     *
     * @return int
     */
    public function getFrozen(): int
    {
        return $this->frozen;
    }
}

File: java/src/main/java/com/mbpay/model/BalanceResponse.java
package com.mbpay.model;

/**
 * 余额查询响应
 */
public class BalanceResponse {
    private long balance; // 可用余额（MB）
    private long frozen;  // 冻结余额（MB）

    public BalanceResponse(long balance, long frozen) {
        this.balance = balance;
        this.frozen = frozen;
    }

    public long getBalance() {
        return balance;
    }

    public long getFrozen() {
        return frozen;
    }
}






